src/openzaak/components/catalogi/admin/mixins.py

```diff
@@ -63,17 +63,25 @@ def response_post_save_change(self, request, obj):
             storage = messages.get_messages(request)
             for i in storage:
                 pass
+            # clear for change view
+            if len(storage._loaded_messages) == 1:
```

Comment:
Why do you need it?

Reply:
was needed when rendering a template instead of redirecting.